clevertap-core/src/main/java/com/clevertap/android/sdk/ManifestInfo.java

```diff
@@ -212,6 +234,13 @@ static void changeCredentials(String id, String token, String region) {
         accountRegion = region;
     }
 
+    static void changeCredentials(String id, String token, String _proxyDomain, String _spikyProxyDomain) {
```

Comment:
> Visibility modifier missing.

It is intended to make it package-private. We expose it's usage via CleverTapAPI class.(Facade pattern)


> what if it is based from different region?

We have overloaded ChangeCredentials() method since either region will be passed or proxyDomains, not both:

1) `changeCredentials(String id, String token, String region)`
2) `changeCredentials(String id, String token, String _proxyDomain, String _spikyProxyDomain)`